versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">= 3.40.0"
+      version = ">= 3.64.0"
```

Comment:
there is no need in updating module to minimal required version "3.64", please leave 3.40 so it won't break any already existing fucntionality